<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="AFSWWI9CYZ2H4A" failures="0" tests="4" name="com.developerscorner.provider.service.AuthServiceTest" time="0.020" errors="0" timestamp="2023-01-31T14:06:26 CST" skipped="0">
  <testcase classname="com.developerscorner.provider.service.AuthServiceTest" name="shouldAuthenticateAUser" time="0.008"/>
  <system-out/>
  <testcase classname="com.developerscorner.provider.service.AuthServiceTest" name="shouldThrowUsernameNotFoundExceptionIfUserNotExists" time="0.007"/>
  <system-out/>
  <testcase classname="com.developerscorner.provider.service.AuthServiceTest" name="shouldThrowConflictExceptionIfUserExists" time="0.003"/>
  <system-out/>
  <testcase classname="com.developerscorner.provider.service.AuthServiceTest" name="shouldRegisterAUser" time="0.002"/>
  <system-out/>
</testsuite> <!-- com.developerscorner.provider.service.AuthServiceTest -->
